### PR TITLE
Feature - included question for tests in PR template

### PR DESCRIPTION
**Pull request by naveensrinivasan**
#### What type of PR is this?
To help drive tests into the PR process.

/kind cleanup

#### What this PR does / why we need it:
As the project is growing, it helps communicate to the PR's contributors require automated tests.

#### Which issue(s) this PR fixes:

None

#### Special notes for your reviewer:
NONE
#### Does this PR introduce a user-facing change?
NONE

```release-note
NONE

```

### CODE REVIEW

**Review by saschagrunert — APPROVED** [review]
/lgtm

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -40,6 +40,12 @@ or
 None
 -->
 
+#### Does this PR have test?
+
+<!--
+If tests aren't applicable just write N/A.
+>
```
Comment (saschagrunert) [inline .github/PULL_REQUEST_TEMPLATE.md:47]: This is not a complete HTML comment. 
```suggestion
<!--
If tests aren't applicable just write N/A.
-->
```

**Review by saschagrunert — COMMENTED** [review]
/ok-to-test

**Review by saschagrunert — APPROVED** [review]
/lgtm

**Comment by naveensrinivasan** [comment]
> /lgtm

@saschagrunert Is the `hold` still required on the PR?

**Comment by saschagrunert** [comment]
/hold cancel
forgot about that, sorry

**Comment by naveensrinivasan** [comment]
> /hold cancel
> forgot about that, sorry

Thanks 😄

**Pull request merged**